### ISSUE TITLE
Option to disable fade

### ISSUE THREAD

**Issue by SkedgyEdgy** [Fugiman/discord-reactive-images#71]
The fading between two different avatars whenever I speak is just... ugly. It should have the ability to be disabled, and the fact that it doesn't even transition properly and makes the image transparent for a half second just makes it worse. PNGTubers should have some level of snappiness to them when they speak, especially if you're using the bounce effect, but this fade just makes it feel extremely slow and bland.

**Comment by niconiconiivt** [comment]
I'd also like the option to disable the fade effect if possible, for those who hope to use such a feature - I agree that it's a lot snappier with no fade + bounce, and I believe reactive images has the function to override the image displayed for other people in the call (if it's really problematic?)

I remember Fugiman stating the reasoning behind the fade in another thread, but I do hope fade disable is implemented someday (maybe as an experimental feature?). Thank you for considering this request!
 
"The opacity styling is required as there is no guarantee that the speaking & inactive PNGs are the same size/shape. Some people may put an outline on the speaking image, or have a different pose entirely. The current code does a 200ms cross-fade between the two, so yes at a certain point both images are at 50% opacity and you can see what's behind them."
https://github.com/Fugiman/discord-reactive-images/issues/47#issuecomment-829844197

**Comment by SkedgyEdgy** [comment]
Yeah I don't understand why "people could do this" means it has to be required. Even if I did put a glow around my talk sprite it would *still* look better without the fade 'cause of how poorly it handles the transition. Even when the two pngs are different sizes or shapes that still doesn't change anything. In fact it just makes it worse, 'cause I tilt my avatar's head slightly when they talk and have them look away from the "camera" when they're idle. It looks significantly worse with a fade than it does when using the Discord overlay PNGTubers.
